main.tf

```diff
@@ -10,22 +10,17 @@ resource "azurerm_mssql_server" "azuresqldbsrv" {
 }
 
 resource "azurerm_mssql_database" "azuresqldb" {
-  name                             = "${lower(var.project)}${lower(var.stage)}db${var.suffix}"
-  server_id                        = azurerm_mssql_server.azuresqldbsrv.id
-  location                         = var.location
-  resource_group_name              = var.resource_group
-  server_name                      = azurerm_mssql_server.azuresqldbsrv.name
-  edition                          = var.edition
-  requested_service_objective_name = var.performance_class
+  name      = "${lower(var.project)}${lower(var.stage)}db${var.suffix}"
```

Comment:
is it possible to pass the whole string to the lower function here?

Reply:
done